acme/acmego/main.go

```diff
@@ -41,13 +41,36 @@ import (
 
 var gofmt = flag.Bool("f", false, "format the entire file after Put")
 
-var formatters = map[string][]string{
-	".go": []string{"goimports"},
+type formatter struct {
+	cmd     []string
+	enabled func(string) bool
```

Comment:
I suppose we could have a method `Enabled()` which returns true if this callback field is nil.